data/addressbook.json

```diff
@@ -1,45 +1,40 @@
 {
   "persons" : [ {
     "name" : "Alex Yeoh",
-    "phone" : "87438807",
     "email" : "alexyeoh@example.com",
     "address" : "Blk 30 Geylang Street 29, #06-40",
     "remark" : "$remark is gone!",
     "tagged" : [ "friends" ]
   }, {
     "name" : "Bernice Yu",
-    "phone" : "99272758",
     "email" : "berniceyu@example.com",
     "address" : "Blk 30 Lorong 3 Serangoon Gardens, #07-18",
     "remark" : "",
     "tagged" : [ "colleagues", "friends" ]
   }, {
     "name" : "Charlotte Oliveiro",
-    "phone" : "93210283",
     "email" : "charlotte@example.com",
     "address" : "Blk 11 Ang Mo Kio Street 74, #11-04",
     "remark" : "",
     "tagged" : [ "neighbours" ]
   }, {
     "name" : "David Li",
-    "phone" : "91031282",
     "email" : "lidavid@example.com",
     "address" : "Blk 436 Serangoon Gardens Street 26, #16-43",
     "remark" : "",
     "tagged" : [ "family" ]
   }, {
     "name" : "Irfan Ibrahim",
-    "phone" : "92492021",
     "email" : "irfan@example.com",
     "address" : "Blk 47 Tampines Street 20, #17-35",
     "remark" : "",
     "tagged" : [ "classmates" ]
   }, {
     "name" : "Roy Balakrishnan",
-    "phone" : "92624417",
     "email" : "royb@example.com",
-    "address" : "Blk 45 Aljunied Street 85, #11-31",
+    "address" : "Blk 46 Aljunied Street 85, #11-31",
     "remark" : "",
     "tagged" : [ "colleagues" ]
   } ]
+
 }
```

Comment:
I think should leave a line after the end of the .json file if not gradlew checkStyle will show an error which may cause the TravisCI build to fail